peddy/v0.4.2/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:16.04
+
+LABEL maintainer="Alex Sickler (sicklera@email.chop.edu)"
+
+ENV PEDDY_VERSION v0.4.2
+
+RUN apt-get update && apt upgrade -y \
+&&  apt-get install -y python git build-essential libbamtools-dev python-pip python-dev build-essential libssl-dev \
+&& pip install backports.functools-lru-cache==1.5 numpy==1.14.5 click==6.7 coloredlogs==10.0 cycler==0.10.0 \
+Cython==0.28.5 decorator==4.3.0 humanfriendly==4.16.1 kiwisolver==1.0.1 \
+matplotlib==2.2.3 monotonic==1.5 networkx==2.1 pandas==0.23.4 pyparsing==2.2.0 \
+python-dateutil==2.7.3 pytz==2018.5 scikit-learn==0.19.2 scipy==1.1.0 seaborn==0.9.0 six==1.11.0 subprocess32==3.5.2 toolshed==0.4.6 \
+&& pip install cyvcf2==0.8.9 \
```

Comment:
Why is this pip install happening on its own?

Reply:
When trying to install cyvcf2 on it's own, it can't find numpy, tries to install it, and gets an error that Python >= 3.7 is required.
```
Collecting cyvcf2==0.8.9
  Downloading https://files.pythonhosted.org/packages/e4/42/f2016a5d158dc0ab205929d74d510214c8845557d4cd8ab366dcfcb6a3da/cyvcf2-0.8.9.tar.gz (1.2MB)
    Complete output from command python setup.py egg_info:
    Collecting cython
      Downloading https://files.pythonhosted.org/packages/59/c1/0b69d125ab9819869cffff2f416158acf2684bdb4bf54eccf887717e2cbd/Cython-0.29.21-cp27-cp27mu-manylinux1_x86_64.whl (1.9MB)
    Collecting numpy
      Downloading https://files.pythonhosted.org/packages/d2/48/f445be426ccd9b2fb64155ac6730c7212358882e589cd3717477d739d9ff/numpy-1.20.1.zip (7.8MB)
        Complete output from command python setup.py egg_info:
        Traceback (most recent call last):
          File "<string>", line 1, in <module>
          File "/tmp/pip-build-AVg8eC/numpy/setup.py", line 30, in <module>
            raise RuntimeError("Python version >= 3.7 required.")
        RuntimeError: Python version >= 3.7 required.

        ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-AVg8eC/numpy/
    You are using pip version 8.1.1, however version 21.0.1 is available.
    You should consider upgrading via the 'pip install --upgrade pip' command.
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-iDV6DD/cyvcf2/setup.py", line 63, in <module>
        import numpy as np
    ImportError: No module named numpy
 ```